docs/sources/features/explore/index.md

```diff
@@ -274,3 +274,10 @@ Simply clicking the button will return you to the origin dashboard, or, if you'd
 the arrow next to the button to reveal a "Return to panel with changes" menu item.
 
 {{< docs-imagebox img="/img/docs/v64/explore_return_dropdown.png" class="docs-image--no-shadow" caption="Screenshot of the expanded explore return dropdown" >}}
+
+## Query Inspector
+
+To help with debugging queries, Explore allows users to investigate query requests and responses, as well as query statistics,
```

Comment:
```suggestion
To help with debugging queries, Explore allows you to investigate query requests and responses, as well as query statistics,
```

Reply:
Per the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), write in the second person imperative.